docs/source/reference/policies.rst

```diff
@@ -117,6 +117,11 @@ Other supported coordination backends include:
 * PostgreSQL
 * file (for testing when all the services are running on a single host)
 
+StackStorm utilizes the ``OpenStack Tooz`` library for communicating with the
+coordination backend. The coordination backend must support the ``Locking``
+functionality as defined by the ``Tooz`` interface. Please refrence the
+`OpenStack Tooz compatability page <https://docs.openstack.org/tooz/latest/user/compatibility.html>`_
+for more information what interfaces are implemented by various backends.
```

Comment:
Line break here probably wouldn't be a good idea.

Reply:
Sorry, a little confused. What do you want changed?

Reply:
Just a line break to separate those two paragraphs, imo.